kernel/search/plugins/ezsearchengine/ezsearchengine.php

```diff
@@ -96,7 +96,11 @@ public function addObject( $contentObject, $commit = true )
                     // Split text on whitespace
                     if ( is_numeric( trim( $text ) ) )
                     {
-                        $integerValue = (int) $text;
+                        //verify if value does not exceed MySQL int(11) unsigned maximum value
```

Comment:
Based on magic number one line below, I think that you have on your mind the maximum **signed** value of `int`. MySQL documentation says (https://dev.mysql.com/doc/refman/5.7/en/integer-types.html) that maximum unsigned value of int is `4294967295` 😉